### PR TITLE
Fix battery status when battery is conected on AC

### PR DESCRIPTION
**Pull request by trinaldi**
One thing that was missing on my end was the battery block showing `Err` when my laptop was plugged in. I'll mark this as WIP because now I can't get the values back (e.g: removing the AC plug).

[`systemstat` does have a way to verify it](https://github.com/valpackett/systemstat/blob/trunk/src/platform/linux.rs#L624), but it's not working for me:

### CODE REVIEW

**Comment by trinaldi** [comment]
Finally made the function work as expected. When the Battery is at 100% and AC is on, it show `err` for a split of a second.

So, with this PR, when the AC power is on, the program notify it, when you remove it, it shows the remaining time.

File: src/app.rs
Comment (olivesdad) [inline src/app.rs:1]: Nice find!

**Review by olivesdad — APPROVED** [review]
LGTM Thank you!

File: src/app.rs
Comment (trinaldi) [reply src/app.rs:1]: 🚀

**Pull request merged**